Big-Spender/readme.md

```diff
@@ -112,7 +122,9 @@ INSERT YOUR QUERY HERE
 **You:** Then you need an extra clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+CREATE: SELECT SUM(amount) AS total_amount
+FROM spends
+WHERE date = '2021-03-01' OR date = '2021-04-01'
 ```
 
 **Farnoosh:** Fantastic. One last thing, looks like we missed something. Can we add a new transaction to the spends table with a description of 'Computer Hardware Dell' and an amount of £32,000?
```

Comment:
Looks like you missed this last question. How would you insert a new row into the transaction?